File: api/subscribe.php
<?php
require __DIR__ . '/vendor/autoload.php';
require "controller/LeituraController.php";

use Workerman\Worker;

$worker = new Worker();

$worker->onWorkerStart = function(){
    $mqtt = new Workerman\Mqtt\Client('mqtt://broker.hivemq.com:1883');
    
    $mqtt->onConnect = function($mqtt) {
        $mqtt->subscribe('mobg/#', ["qos" => 0]);
    };
    
    $mqtt->onMessage = function($topic, $content){
        echo "\nTópico: " . $topic;
        echo "\nConteúdo: " . $content;

        //$controller = new LeituraController();

        //$conteudo = new \stdClass();
        //$conteudo->sensor = explode('/', $topic)[1];
        //$conteudo->valor = $content;

        //$controller->store($conteudo);
    };

    $mqtt->connect();
};

Worker::runAll();

File: api/subscribe.py
import paho.mqtt.client as mqtt


def on_connect(client, userdata, flags, rc):  
    print("Conectado com o código de retorno {0}".format(str(rc)))  
    client.subscribe("mobg/#") 


def on_message(client, userdata, msg):  
    print("Mensagem recebida -> " + msg.topic + " " + str(msg.payload)) 


client = mqtt.Client("mobg") 
client.on_connect = on_connect  
client.on_message = on_message  
client.connect('test.mosquitto.org', 1883, 17300)
client.loop_forever()  